.github/workflows/dependabot.yml

```
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "gradle"
+    directory: "/"
```

Issue/Suggestion: Target the MPL's Java directory
```suggestion
    directory: "AwsCryptographicMaterialProviders/runtimes/java"
```